src/components/Book/Book.tsx

```diff
@@ -1,15 +1,22 @@
-import { useState } from "react";
 import "./Book.scss";
 import { BookType } from "../BookList/BookList";
 
-export const Book = ({id, title, isbn, authors}: BookType) => {
+export const Book = ({ id, title, isbn, authors }: BookType) => {
   return (
     <div className="book">
-      <img className="image" src="./book-cover-placeholder.png" alt={title}></img>
+      <img
+        className="image"
+        src="./book-cover-placeholder.png"
+        alt={title}
+      ></img>
       <div className="title">{title}</div>
       <ul className="authors">
-      {authors &&
-        authors.map((author) => <li className="author">{author}</li>)}
+        {authors &&
```

Comment:
this statement is always true because authors is an array, you should check authors.length